code/src/layouts/app-banner/AppBanner.tsx

```diff
@@ -1,40 +1,50 @@
-import PageWrapper from "@/layouts/page-wrapper/PageWrapper";
 import AppTypography from "@/components/app-typography/AppTypography";
+import AppContainer from "@/components/app-container/AppContainer";
 import AppButton from "@/components/app-button/AppButton";
 import AppBox from "@/components/app-box/AppBox";
 
-import heroImage from "@/assets/images/home-page/hero_section_img.jpg";
+import useAppBanner from "@/layouts/app-banner/useAppBanner";
+
+import heroImage1 from "@/assets/images/home-page/hero-section-img-1.webp";
+import heroImage2 from "@/assets/images/home-page/hero-section-img-2.webp";
+import heroImage3 from "@/assets/images/home-page/hero-section-img-3.webp";
 
 import "@/layouts/app-banner/AppBanner.scss";
 
+const images = [heroImage1, heroImage2, heroImage3];
+
 const AppBanner = () => {
+  const currentImageIndex = useAppBanner(images, 6);
+
   return (
     <AppBox
       className="spa-banner"
       style={{
-        backgroundImage: `url(${heroImage})`
+        backgroundImage: `url(${images[currentImageIndex]})`
       }}
     >
-      <PageWrapper>
-        <AppBox className="spa-banner__container">
-          <AppBox>
-            <AppTypography
-              className="spa-banner__header"
-              variant="h1"
-              translationKey="appBanner.header"
-            />
-            <AppBox className="spa-banner__paragraph">
-              <AppTypography
-                variant="subtitle2"
-                translationKey="appBanner.paragraphfirst"
-              />
-            </AppBox>
-            <AppBox className="spa-banner__button">
-              <AppButton size="large">Shop Now</AppButton>
-            </AppBox>
-          </AppBox>
+      <AppContainer maxWidth="lg">
+        <AppBox className="spa-banner__wrapper">
+          <div className="best-prices">
```

Comment:
Why do you use <div> instead <AppBox>

Reply:
PR was in drafts, so it was still in progress